sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1,3090 +1,3131 @@
-            },
-            {
-              "name": "x-ms-max-item-count",
-              "in": "header",
-              "required": false,
-              "type": "string"
-            }
-          ],
-          "responses": {
-            "200": {
-              "description": "Query result with continuation token if appropriate.",
-              "schema": {
-                "type": "array",
-                "items": {
-                  "$ref": "#/definitions/Twin"
-                }
+            "headers": {
+              "x-ms-item-type": {
+                "description": "Type of the list of items.",
+                "type": "string"
               },
-              "headers": {
-                "x-ms-item-type": {
-                  "description": "Type of the list of items.",
-                  "type": "string"
-                },
-                "x-ms-continuation": {
-                  "description": "Continuation token",
-                  "type": "string"
-                }
+              "x-ms-continuation": {
+                "description": "Continuation token",
+                "type": "string"
               }
             }
           }
         }
-      },
-      "/devices/{id}": {
-        "get": {
-          "summary": "Retrieve a device from the identity registry of an IoT hub.",
-          "description": "Retrieve a device from the identity registry of an IoT hub. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-          "operationId": "RegistryManager_GetDevice",
-          "consumes": [],
-          "produces": [
-            "application/json"
-          ],
-          "parameters": [
-            {
-              "name": "id",
-              "in": "path",
-              "description": "Device ID.",
-              "required": true,
-              "type": "string"
-            },
-            {
-              "$ref": "#/parameters/api-version"
-            }
-          ],
-          "responses": {
-            "200": {
-              "description": "Returns the Device object",
-              "schema": {
-                "$ref": "#/definitions/Device"
-              }
+      }
+    },
+    "/devices/{id}": {
+      "get": {
+        "description": "Retrieve a device from the identity registry of an IoT hub.",
+        "operationId": "RegistryManager_GetDevice",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique identifier of the device to retrieve.",
```

Comment:
Wouldn't it be better to highlight that this is the device ID. We don't really refer to device IDs are "unique identifier of the device" in other documents/ readme etc, do we?

Reply:
I'll keep it, since this wording was asked for by Priyanka above

Reply:
Ah, got it!